### PR TITLE
Refine desktop location fetch

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

Undoes #984 and instead, displays an alert on desktop when attempting to use the current location for the first time after loading the page, and location access is blocked.

i18n messages are provided when the user denied location access.

## PR Checklist
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/actions/location.tsx [hunk 1]
```diff
@@ -47,9 +48,28 @@ export function getCurrentPosition(
         // On error
         (error) => {
           console.log('error getting current position', error)
-          // FIXME, analyze error code to produce better error message.
-          // See https://developer.mozilla.org/en-US/docs/Web/API/GeolocationPositionError
-          dispatch(receivedPositionError({ error }))
+          // On desktop, after user clicks "Use location" from the location fields,
+          // show an alert and explain if location is blocked.
+          if (!isMobile() && error.code === 1) {
+            window.alert(
```
Comment (miles-grant-ibigroup) [inline lib/actions/location.tsx:54]: A bit clunky, but I think good enough for now! Could we possibly add a TODO here to move this code into `location-field` and handle this more elgantly

**Review by miles-grant-ibigroup — APPROVED** [review]
Much better thank you!

File: lib/actions/location.tsx (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/actions/location.tsx:55]: I added the TODO in https://github.com/opentripplanner/otp-react-redux/pull/989/commits/933d34a0c8fc1838653a27433651b5e21b92c218.

**Review by philip-cline — APPROVED** [review]
LGTM

**Pull request merged**